DB/DB_SCHEMA.sql

```diff
@@ -0,0 +1,28 @@
+create type category as enum('PHONE','LAPTOP','DESKTOP','AUDIO','TABLET','EARPHONE');
+
+CREATE TABLE PRDT_INFO(
+    PRDT_INFO_ID SERIAL PRIMARY KEY,
+    PRDT_NAME VARCHAR(255) NOT NULL UNIQUE,
+    PRDT_DESC TEXT,
+    PRDT_TYPE category ,
```

Comment:
Use `PRDT` prefix is slightly redundant.

```sql
SELECT PRDT_INFO.PRDT_NAME FROM PRDT_INFO;
```

The column naming should be short and clear.

```sql
SELECT PRDT_INFO.NAME FROM PRDT_INFO;
```

Reply:
IIUC, it should not be an issue.

You cannot use the same column `name` in two or more tables within a single SQL query. This will result in an error in the DB.

Instead, you should use the explict table name, like `A.name` and `B.name`.

Reply:
agree with not using prefixes.

Reply:
As I know, it's could be an issue.
In my job,I use ORACLE DB.
If I have a SQL Command like this:
```sql
SELECT A.cust_id, A.cust_name, B.cust_name, B.order_no
FROM customers A
JOIN orders B ON A.cust_id = B.cust_id
WHERE cust_id = 'XXXXX'

this SQL won't throws error exception.
But if I use cust_name behind `ON` statement, it's will throws.
So use explict table name could fix error in `JOIN`
But I'm talk about `SELECT` keyword.